### PR TITLE
hwdb update + contrib list

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM.

**Comment by poettering** [comment]
lgtm

**Review by bluca — APPROVED** [review]
lgtm too

**Comment by poettering** [comment]
the archlinux/centos failure is something with numa, almost cetrainly unrelated

File: NEWS [hunk 1]
```diff
@@ -440,39 +440,43 @@ CHANGES WITH 248:
...
+        Sergey Bugaev, shenyangyang4, simmon, Simonas Kazlauskas,
+        Slimane Selyan Amiri, Stefan Agner, Steve Ramage, Susant Sahani,
+        Sven Mueller, Tad Fisher, Takashi Iwai, Thomas Haller, Tom Shield,
+        Topi Miettinen, Torsten Hilbrich, tpgxyz, Tyler Hicks, ulf-f,
+        Ulrich Ölmann, Vincent Pelletier, Vinnie Magro, Vito Caputo, Vlad,
+        walbit-de, Whired Planck, wouter bolsterlee, Xℹ Ruoyao, Yangyang Shen,
+        Yuri Chornoivan, Yu Watanabe, Zach Smith, Zbigniew Jędrzejewski-Szmek,
+        Zmicer Turok, Дамјан Георгиевски
 
         — Berlin, 2021-03-17
```
Comment (yuwata) [inline NEWS:481]: (Please update the release date before tagging v248.)

**Comment by bluca** [comment]
centos failure is testbed fluke, bionic-* is all https://github.com/systemd/systemd/issues/19154

**Pull request merged**